moveit_core/robot_state/include/moveit/robot_state/attached_body.h

```diff
@@ -153,12 +156,8 @@ class AttachedBody
   /** \brief Set the scale for the shapes of this attached object */
   void setScale(double scale);
 
-  /** \brief Recompute global_collision_body_transform given the transform of the parent link*/
-  void computeTransform(const Eigen::Isometry3d& parent_link_global_transform)
-  {
-    for (std::size_t i = 0; i < global_collision_body_transforms_.size(); ++i)
-      global_collision_body_transforms_[i] = parent_link_global_transform * attach_trans_[i];
-  }
+  /** \brief Recompute global_collision_body_transform given the transform of the parent link */
```

Comment:
The original description is kind of outdated. The function recomputes `global_collision_body_transforms`  (plural) and those of the subframes now. How about changing the description like this:

"Recompute global_collision_body_transforms and global_subframe_poses given the transform of the parent link "

And maybe even rename to "updateGlobalTransforms".